File: Nov 1/Basic/q7.cpp
// Write a program that takes minutes as input, and display the total number of hours and minutes. (4)

#include <iostream>
using namespace std;

int main() {
    int minutes, hours;

    cout << "Enter the minutes: " << endl;
    cin >> minutes;

    hours = minutes / 60;
    minutes = minutes % 60;

    cout << hours << " Hours ";
    if (minutes !=0) {
            cout << minutes << " Minutes";
    }
    cout << endl;

    return 0;
}

File: Nov 1/Basic/q7.java
// Write a program that takes minutes as input, and display the total number of hours and minutes. (4)

import java.util.*;

class q7 {
    public static void main(String args[]) {
        Scanner s = new Scanner(System.in);
        int minutes, hours;

        System.out.println("Enter the minutes: ");
        minutes = s.nextInt();

        hours = minutes / 60;
        minutes = minutes % 60;

        System.out.print(hours + " Hours ");
        if (minutes !=0) {
            System.out.print( minutes + " Minutes");
        }
        System.out.println();
        s.close();
    }
}